Melhorando tempo de instalação do quoti-cli

luizAlbuquerqueJr opened the pull request:
https://www.notion.so/beyondco/Tasks-Quoti-Roadmap-bed7b1e542c045c2ac484732c9e60bc4?p=3afe4efc72c94264afc5887f840841db

Commit:
build: add CLI-deploy/node_modules and CLI-deploy/src to gitignore

Commit:
build: add script build and postInstall

Commit:
feat: add necessary file to cli-deploy

Commit:
build: add qt.7z to gitignore

nmf2 committed:
fix: remove deploy command

nmf2 committed:
build: add README.md

nmf2 committed:
build: add package.json with version

nmf2 committed:
0.7.0-beta.1

nmf2 committed:
0.7.0-beta.2

ovictoraurelio commented on .gitignore:
```diff
@@ -12,4 +12,7 @@ test/beyondCredentials.json
-.eslintcache
\ No newline at end of file
+.eslintcache
+CLI-deploy/node_modules
+CLI-deploy/src
+CLI-deploy/qt.7z
```
Por que esses caras tão no .gitignore mas ainda assim estão aqui no git também? 

Se não é para estarem mais no git podes fazer:

$ git rm CLI-deploy/

nmf2 replied:
@ovictoraurelio, solved!

nmf2 committed:
chore: add build and publish pipeline

nmf2 committed:
fix: serve test

nmf2 committed:
0.7.0

nmf2 committed:
build: finish publish script

nmf2 committed:
0.8.0-beta.1

nmf2 committed:
build: there needs to be a dummy script for the bin property of the package.json

nmf2 committed:
0.8.0-beta.2

nmf2 committed:
build: add node version check

nmf2 committed:
ci: improve publish script to support tags

The publis script now automatically reads any tag
from the version and publishes the package to that
tag on npm. e.g. If the version is 0.10.0-beta.0
the publish command to be executed will be
"npm publish --tag beta"

nmf2 committed:
ci: remove devDeps optimization

nmf2 committed:
dry publish run

nmf2 committed:
ci: fix script to get tag to publish

nmf2 committed:
0.8.0-beta.4

nmf2 committed:
0.8.0-beta.5

nmf2 committed:
fix: remove extra publish command

nmf2 committed:
0.8.0-beta.6
